compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -2585,8 +2585,7 @@ TR::Register *OMR::X86::TreeEvaluator::overflowCHKEvaluator(TR::Node *node, TR::
    }
 
 extern "C" void *fwdHalfWordCopyTable;
-
-TR::Register *OMR::X86::TreeEvaluator::arraycopyEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+static TR::Register * deprecated_arraycopyEvaluator(TR::Node *node, TR::CodeGenerator *cg)
```

Comment:
The changeset to delete deprecated code is ready and I will deliver it in a subsequent PR 3-4 days after this PR gets merged. I would like to have this version on record so that other developers on OMR could have a chance to see the changes on ArrayCopy evaluator.